tests/slicing/test-runner.py

```diff
@@ -91,6 +91,7 @@ def compile(source, output=None, params=[]):
         output = _getbcname(source)
 
     ret = command(["clang", "-include", join(SOURCESDIR, '..', "test_assert.h"),
+                   "-fno-stack-protector", "-D_FORTIFY_SOURCE=0",
```

Comment:
Shouldn't we rather specify that `__memcpy_chk` behaves like `memcpy` to fix the tests? Or what was the real problem here?

Reply:
Sure, that seems more reasonable.

Reply:
I just pushed two commits that should resolve this conversation into the `memcpy_chk` branch. Could you check that it works (once you have the new laptop, of course).

Reply:
Yes, these changes work as well (at least according to CI: https://github.com/lzaoral/dg/actions/runs/512460855). Thanks!